_overviews/scala3-book/taste-control-structures.md

```diff
@@ -45,7 +45,17 @@ As you’ll see throughout this book, _all_ Scala control structures can be used
 > An expression returns a result, while a statement does not.
 > Statements are typically used for their side-effects, such as using `println` to print to the console.
 
+The `if`/`else` control struture in Scala 2 was constructed differently, with parentheses required and curly brackets instead of the keyword `then`.
 
+```scala
+if (test1) {
```

Comment:
What do you think of repeating, in the code snippet, that we are using Scala 2 syntax?

```suggestion
// Scala 2 syntax
if (test1) {
```